### PR TITLE
Add documentation for DECIMAL casts

### PR DESCRIPTION
**Pull request by losipiuk**
Add documentation for DECIMAL casts

### CODE REVIEW

File: presto-docs/src/main/sphinx/functions/decimal.rst [hunk 1]
```diff
@@ -63,3 +63,14 @@ Unary Decimal Operators
 -----------------------
 
     The ``-`` operator performs negation. The type of result is same as type of argument.
+
+Casts
+-----
+
+    Following type cast are currently supported for ``DECIMAL`` values:
```
Comment (akshatnair) [inline presto-docs/src/main/sphinx/functions/decimal.rst]: s/cast/casts

File: presto-docs/src/main/sphinx/language/decimal_limitations.rst [hunk 2]
```diff
@@ -0,0 +1,15 @@
+
+=============================
+Decimal data type limitations
+=============================
+
+ #. Currently only basic arithmetic operators and inequality operators are supported.
+
+ #. Automatic coercions are only supported between ``DECIMAL`` types (where possible).
+    Other supported casts must be specific explicitly by the user is SQL.
```
Comment (akshatnair) [inline presto-docs/src/main/sphinx/language/decimal_limitations.rst]: s/specific/specified

**Comment by akshatnair** [comment]
lgtm mod comments

File: presto-docs/src/main/sphinx/language/decimal_limitations.rst [hunk 3]
```diff
@@ -0,0 +1,15 @@
+
+=============================
+Decimal data type limitations
+=============================
+
+ #. Currently only basic arithmetic operators and inequality operators are supported.
```
Comment (kbajda) [inline presto-docs/src/main/sphinx/language/decimal_limitations.rst:6]: Could you please mention explicitly a few things that are NOT supported? E.g., does SELECT SUM(decimal_col) FROM decimal_table WHERE ROUND(decimal_col)>10 work?

**Pull request closed without merging**